packages/vats/src/core/client-behaviors.js

```diff
@@ -80,7 +80,7 @@ async function createLocalBundle(vats, devices, vatAdminSvc, vatPowers) {
     http: httpRegCallback,
     vattp: makeVattpFrom(vats),
   });
-  /** @type { typeof bundleP } */
+  /** @type {Promise<typeof bundleP>} */
```

Comment:
using `deeplyFulfilledObject` might work better here and not require a type annotation

Reply:
Thanks!  That works well.